specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ContainerApps.json

```diff
@@ -489,6 +546,13 @@
             "revisionName"
           ]
         },
+        "customDomains": {
+          "description": "custom domain bindings for Container Apps' hostnames.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CustomDomain"
+          }
```

Comment:
For arrays you may have to add this:

"x-ms-identifiers": [
            "name"
          ]

Reply:
added `x-ms-identifiers`